src/Management/test/EndpointCore.Test/Health/EndpointMiddlewareTest.cs

```diff
@@ -149,6 +149,32 @@ public async Task HealthActuator_ReturnsDetails()
             Assert.Contains("diskSpace", health["details"].ToString());
         }
 
+        [Fact]
+        public async Task HealthActuatorV3_ReturnsDetails()
+        {
+            var settings = new Dictionary<string, string>(appSettings) { { "management:endpoints:customjsonconverters:0", typeof(HealthConverterV3).FullName} };
```

Comment:
So if I want my app to use the v3 health response, I need to use app settings (as expected) - is this what it looks like in pure json?
```json
{
  "Management": {
    "Endpoints": {
      "CustomJsonConverters": [
        "Steeltoe.Management.Endpoint.Health.HealthConverterV3"
      ]
    }
  }
}
```

Reply:
Thinking about this from the perspective of needing additional response changes very soon (mappings at least, right?) do we want to add a shorthand property for something like "preferred response version" that can default to v2 here but be v3 in Steeltoe 4.0 so that each converter wouldn't need to be specified? Maybe excessive, but it would be nicer to have one small change than need to copy/paste `n` converters in here if it's not a huge deal

Reply:
I am of the thought that we best do that in  v4 that doesn't need backwards compatibility. In 3.2, adding any additional properties to this interface (IManagementOptions) is a breaking change. It's a major refactor to get this option into where in the code its needed - lots of public API that would need changed - so this is what I finally came up with.

Reply:
Ok on doing it as-is and making it easier in v4 - is the json I typed out accurate?

Reply:
Yes the json is correct.